server/src/common/redis/redis.module.ts

```diff
@@ -0,0 +1,32 @@
+import { Global, Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { createClient, RedisClientType } from 'redis';
+
+export const REDIS_CLIENT = 'REDIS_CLIENT' as const;
+
+@Global()
+@Module({
+  imports: [ConfigModule],
+  providers: [
+    {
+      provide: REDIS_CLIENT,
+      useFactory: async (

```

Comment:
```suggestion
      useFactory,
```
이 부분의 익명 함수가 다른 `key: value` 들과 섞여있어서 개인적으로 읽기가 어려웠습니다!
useFactory라는 함수형 변수를 분리하고 여기에는 값을 주기만 하면 어떨까 생각됩니다!

Reply:
분리했습니다!